File: server/projects/profile/src/main/java/com/raxim/myscoutee/profile/data/dto/rest/LinkInfo.java
package com.raxim.myscoutee.profile.data.dto.rest;

import com.fasterxml.jackson.annotation.JsonProperty;

public class LinkInfo {
    @JsonProperty(value = "title")
    private String title;

    @JsonProperty(value = "msg")
    private String msg;

    public LinkInfo(String title, String msg) {
        this.title = title;
        this.msg = msg;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getMsg() {
        return msg;
    }

    public void setMsg(String msg) {
        this.msg = msg;
    }
}

File: server/projects/profile/src/main/java/com/raxim/myscoutee/profile/data/dto/rest/LinkInfo.kt
package com.raxim.myscoutee.profile.data.dto.rest

import com.fasterxml.jackson.annotation.JsonProperty

data class LinkInfo(
    @field:JsonProperty(value = "title")
    val title: String? = null,
    @field:JsonProperty(value = "msg")
    val msg: String? = null
)